src/editor.js

```diff
@@ -44,16 +38,28 @@ CKEDITOR.define( [
 			 * global configurations available in {@link CKEDITOR.config} if configurations are not found in the
 			 * instance itself.
 			 *
-			 * @type {Config}
+			 * @readonly
+			 * @type {Config} config
 			 */
-			this.config = new EditorConfig( config );
+			this.set( 'config', new EditorConfig() );
 
 			/**
 			 * The plugins loaded and in use by this editor instance.
 			 *
-			 * @type {PluginCollection}
+			 * @readonly
+			 * @type {PluginCollection} plugins
 			 */
-			this.plugins = new PluginCollection( this );
+			this.set( 'plugins', new PluginCollection( this ) );
+
+			/**
+			 * The editables attached to this editor instance.
+			 *
+			 * @readonly
+			 * @type {EditableCollection} editables
+			 */
+			this.set( 'editables', new EditableCollection() );
+
+			this._creators = {};
```

Comment:
Shouldn't `_creators` be a CREDITOR property? The editor constructor only takes one and use it. There is no point in registering creator for every editor instance.